src/main/java/cms/gov/madie/terminology/webclient/TerminologyServiceWebClient.java

```diff
@@ -111,4 +122,28 @@ public VsacCode getCode(String codePath, String serviceTicket) {
             })
         .block();
   }
+
+  public String getTgt(String apiKey) throws InterruptedException, ExecutionException {
+    String uri = String.format(baseUrl + utsLoginEndpoint, apiKey);
+    Mono<String> responseMono =
+        terminologyClient
+            .post()
+            .uri(uri)
```

Comment:
VSAC and UMLS recommends sending the api-key in the request body. It looks it is being sent in the URL. Is that correct?